universal-application-tool-0.0.1/app/controllers/ti/TrustedIntermediaryController.java

```diff
@@ -115,10 +119,13 @@ public Result addApplicant(Long id, Http.Request request) {
       return redirect(
           routes.TrustedIntermediaryController.dashboard(Optional.empty(), Optional.empty()));
     } catch (EmailAddressExistsException e) {
+      String tiUrl = baseUrl + "/trustedIntermediaries";
```

Comment:
readability nit: Use descriptive names to be clear to a new reader. I would change "tiURL" to "trustedIntermediaryUrl"

Reply:
Done